File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/Creating2DBarcode.php
<?php
namespace Aspose\Barcode\WorkingWith2DBarcodes\Basic2DBarcodeFeatures;

use com\aspose\barcode\BarCodeBuilder as BarCodeBuilder;
use com\aspose\barcode\Symbology as Symbology;

class Creating2DBarcode{

    public static function run($dataDir=null){

        # Instantiate barcode object
        $builder = new BarCodeBuilder();

        $symbology=new Symbology();
        $builder->setSymbologyType($symbology->Pdf417);

        # Width of each module
        $builder->setxDimension(0.6);

        # Height of each module
        $builder->setyDimension(1.2);
        $builder->setCodeText("this is some test code text. \n Second line \n third line.");

        # Save the image to your system and set its image format to Jpeg
        $builder->save($dataDir . "Creating2DBarcode.jpg");

        # Display Status
        print "Created 2D Barcode Successfully.".PHP_EOL;
    }
}

File: Plugins/Aspose.BarCode Java for Jython/asposebarcode/WorkingWith2DBarcodes/Basic2DBarcodeFeatures/Creating2DBarcode.py
from asposebarcode import Settings
from com.aspose.barcode import BarCodeBuilder
from com.aspose.barcode import Symbology

class Creating2DBarcode:

    def __init__(self):
        
        dataDir = Settings.dataDir + 'WorkingWith2DBarcodes/Basic2DBarcodeFeatures/Creating2DBarcode'
        
        # Instantiate barcode object
        builder =  BarCodeBuilder()

        symbology= Symbology
        builder.setSymbologyType(symbology.Pdf417)

        # Width of each module
        builder.setxDimension(0.6)

        # Height of each module
        builder.setyDimension(1.2)
        builder.setCodeText("this is some test code text. \n Second line \n third line.")

        # Save the image to your system and set its image format to Jpeg
        builder.save(dataDir + "Creating2DBarcode.jpg")

        # Display Status
        print "Created 2D Barcode Successfully."

if __name__ == '__main__':        
    Creating2DBarcode()